edgelet/edgelet-core/src/module.rs

```diff
@@ -366,34 +365,13 @@ pub struct SystemInfo {
     pub architecture: String,
     /// iotedge version string
     pub version: &'static str,
-    pub provisioning: ProvisioningInfo,
     pub server_version: String,
     pub kernel_version: String,
     pub operating_system: String,
     pub cpus: i32,
     pub virtualized: &'static str,
 }
 
-#[derive(Clone, Debug, Default, Serialize)]
-pub struct ProvisioningInfo {
```

Comment:
@lfitchett - is telemetry the only purpose for querying all the provisioning settings? I ask because these settings have shifted to iot-identity-service and this will translate to adding more properties in the device identity information returned if needed.

Reply:
Yes, it is. @damonbarry actually added this part, but id is just used to fill in metadata metrics. See https://github.com/Azure/iotedge/blob/master/edgelet/edgelet-docker/src/runtime.rs#L322 for its usage.

Reply:
Thanks. It makes sense to emit the real values from Identity Service now. I will work on this in a different PR. Placeholder values will be used in these preview bits for now.